### ISSUE TITLE
[Linux] [MacOS] [Mods] [Urgent?] Almost all mods incompatible with Linux, MacOS

### ISSUE THREAD

**Issue by hubismal** [Regalis11/Barotrauma#1930]
tldr please make linux and macos not case sensitive when trying to find files somehow

MacOS and Linux are case sensitive when it comes to referring to files while the windows version seems to not be case sensitive. Because of this, most mods (which happen to be made on windows) flat out don't work (examples of this are the spec ops, colonial marines, BTE mods), often spilling large amounts of errors about files not existing, and the mod itself fails to load. This results in a large compatibility issue and it's hard for the creators to scope out which files are the offenders because their versions of barotrauma don't notice these uncapitalized files.

It would be extremely nice (and necessary) if the windows version of the game was made to be case sensitive, or the linux/macos versions were made to not be case sensitive. For the sake of clarity, it should probably be the former because I think capitalization matters, but making linux/macos not case sensitive might be more backwards compatible.

**Comment by juanjp600** [comment]
Addressed by https://github.com/Regalis11/Barotrauma-development/commit/f373f27ce1fc1536481fed069872937d0e54ee60

**Comment by Rokvach** [comment]
Confirmed working correctly, moving to Done.
